apps/react-app/src/pods/template-export/template-export.component.tsx

```diff
@@ -1,26 +1,46 @@
-import { Button } from '@/common-app/components';
 import React from 'react';
+import { useUserChoiceContext } from '@/core';
+import { Button, Footer, Header, Navbar } from '@/common-app/components';
+import * as classes from './template-export.styles';
 
 interface Props {
-  onExport: () => void;
+  onExport: (text: string) => void;
 }
 
 export const TemplateExport: React.FC<Props> = props => {
   const { onExport } = props;
+  const { userChoice, setUserChoice } = useUserChoiceContext();
+  const [text, setText] = React.useState<string>('');
 
-  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-    onExport();
+  const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setText(event.target.value);
+  };
+
+  const handleOnExport = () => {
+    if (text.trim().length !== 0) {
```

Comment:
We can remove this condition and disable button if text is equal to empty string